zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -921,18 +912,15 @@ pub(crate) fn spend_succeeds_to_t_addr_zero_change<T: ShieldedPoolTester>() {
     let dfvk = T::test_account_fvk(&st);
 
     // Add funds to the wallet in a single note
-    let value = NonNegativeAmount::const_from_u64(60000);
+    let value = NonNegativeAmount::const_from_u64(70000);
```

Comment:
Is this bump necessary for the test to work under ZIP 317, or is it left over from your other PR (that you subsequently undid)?

Reply:
No, this is necessary because the ZIP 317 fee is higher than 10000 zats. (Actually a `value` of 65000 zats would be sufficient, but this is not intended to be testing the zero-change corner case.)